pumpkin/src/world/mod.rs

```diff
@@ -347,6 +357,16 @@ impl World {
         None
     }
 
+    /// Gets a Player by UUID
+    pub async fn get_player_by_uuid(&self, id: uuid::Uuid) -> Option<Arc<Player>> {
+        for player in self.current_players.lock().await.values() {
+            if player.gameprofile.id == id {
+                return Some(player.clone());
+            }
+        }
+        None
```

Comment:
Its a HashMap, You can search the player by uuid in it

Reply:
Thanks, should be resolved in [7b7eab5](https://github.com/Snowiiii/Pumpkin/pull/178/commits/7b7eab5dc67c47752c52a7b2b6aa1dcf60abf748) 🙂

Reply:
marking as resolved